engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/CreateGameScreen.java

```diff
@@ -91,6 +91,11 @@
 
     private boolean loadingAsServer;
 
+    /**
+     * A UniverseWrapper object used here to determine if the game is single player or multi-player.
```

Comment:
Consistency. Either use `single-player` or `multi player`, but not both - and specially not in the same line.